includes/wikia/api/ArticlesApiController.class.php

```diff
@@ -358,7 +358,7 @@ public function getNew(){
 			$ns = array_unique( $ns );
 		}
 
-		$key = self::getCacheKey( self::NEW_ARTICLES_CACHE_ID, '', [ implode( '-', $ns ) , $minArticleQuality ] );
+		$key = self::getCacheKey( self::NEW_ARTICLES_CACHE_ID, '', [ implode( '-', $ns ) , $minArticleQuality, $limit ] );
```

Comment:
It looks like 80% of the queries are for limit=1 and limit=10 and ~0% on the maximum. Furthermore, we have caching on ImageServing end too (per article). I'd leave it as in this PR to avoid slicing on vast majority of requests.